### PR TITLE
Updates guzzle

### PR DESCRIPTION
**Pull request by CallumEddisford**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "type": "library",
     "require": {
         "php": "^7.1.3",
-        "guzzlehttp/guzzle": "^6.3",
+        "guzzlehttp/guzzle": "^7",
```
Comment (Smudge3806) [inline composer.json:7]: Is Guzzle 7 compatible with Illuminate 6?

Comment (th0rn0) [reply composer.json:7]: just testing some stuff on another branch. This works with --no-scripts but I think there might be another issue with messaging

Comment (th0rn0) [reply composer.json:7]: `  Too few arguments to function Illuminate\Database\Migrations\MigrationCreator::__construct(), 1 passed in /app/vendor/xedi/pdk/src/Formation/Migrations/MigrationServiceProvider.php on line 46 and exactly 2 expected
`

Comment (th0rn0) [reply composer.json:7]: https://github.com/xedi/pdk-php/pull/439

Comment (th0rn0) [reply composer.json]: also push slight update so its ^6.3|^7 for guzzle

**Pull request merged**